services/web-app/utils/schema.js

```diff
@@ -76,6 +72,15 @@ export const getAssetType = (asset) => {
   return get(type, `${asset.content.type}`, type.component)
 }
 
+/**
+ * Gets the asset status string value. If not found, defaults to 'draft'.
+ * @param {import('../typedefs').Asset} asset
+ * @returns {string} Asset status value
+ */
+export const getAssetStatus = (asset) => {
+  return get(asset.content, ['status', 'key'], asset.content.status) ?? 'draft'
```

Comment:
Same question here about the usage of lodash `get`. Is this the same is just doing this?:

```js
return asset.content.status.key || asset.content.status || 'draft'
```

Reply:
yeah same thing except in this case `...`.key could error out and lodash wont

Reply:
done!